src/components/chats/components/CreateNewChat/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { Skeleton } from '@/components/ui/skeleton'
+import { useNavigation } from '@/lib/frontend/useNavigation'
+import { FC } from 'react'
+import { CreateChat } from './CreateChat'
+import { CreateStandaloneChat } from './CreateStandaloneChat'
+import { CreateNewChatProps } from './types'
+
+export const CreateNewChat: FC<CreateNewChatProps> = ({
+  loading,
+  ...props
+}) => {
+  const { query } = useNavigation()
+  const postId = query.post_id
+  const isStandalone = !!postId
```

Comment:
I would avoid relying on the navigation to determine if the post isStandalone. I see it less future proof; if we change it will be hard to remember about this specific logic.

Instead, the component receives `postId`. I'd take the postId and compare it against `useDefaultPost`. If the value of the ids is the same, then `isStandalone === true`.

Reply:
But we are already on the `FE` why rely so much in  the `BE`.  Changing route navigation params I guess should be a major change. If we remove post_if many pages and logic inside the `FE` will have to change.